rmgpy/molecule/group.py

```diff
@@ -908,7 +912,23 @@ def is_hydrogen_bond(self, wildcards=False):
             else:
                 return False
         else:
-            return abs(self.order[0]) <= 1e-9 and len(self.order) == 1
+            return abs(self.order[0] - 0.1) <= 1e-9 and len(self.order) == 1
+        
+    def is_reaction_bond(self, wildcards=False):
+        """
+        Return ``True`` if the bond represents a hydrogen bond or ``False`` if
+        not. If `wildcards` is ``False`` we return False anytime there is more
+        than one bond order, otherwise we return ``True`` if any of the options
+        are hydrogen bonds.
+        """
+        if wildcards:
+            for order in self.order:
+                if abs(order) <= 1e-9:
```

Comment:
I think this was part of an existing bug that I copied over where the H bonds used to be 0 value, but then they were changed to 0.1 and check was still left against order instead of (order - 0.1). I've fixed that issue here now for H and R bonds.